R/pkg/R/DataFrame.R

```diff
@@ -946,6 +973,17 @@ setMethod("write.parquet",
             invisible(handledCallJMethod(write, "parquet", path))
           })
 
+#' @rdname write.parquet
+#' @name saveAsParquetFile
+#' @aliases saveAsParquetFile,SparkDataFrame,character-method
+#' @note saveAsParquetFile since 1.4.0
+setMethod("saveAsParquetFile",
```

Comment:
Are we sure we want to keep this too? Scala and PySpark sides don't have it either as of SPARK-12600 cc @rxin and @marmbrus 

https://github.com/apache/spark/commit/77ab49b8575d2ebd678065fa70b0343d532ab9c2#diff-c3d0394b2fc08fb2842ff0362a5ac6c9

Reply:
`jsonFile` and `parquetFile` too.

Reply:
Okay, using the rubric we added, in R or PySpark, it's very easy to keep the compatibility - if-else based on version and redirect to the alternative. There seems direct alternative too.

Seems like we haven't seen a lot of complaints when we remove `saveAsParquetFile` at  SPARK-12600. I skimmed user-mailing list with keyword search, and I found 0 complaints.

Having write API in `DataFrame` seems inconsistent with Scala and PySpark sides, and also namespace used in R side (e.g., `write.json`) - it causes a cognitive cost.

Shall we remove these one out?

Reply:
OK. If pySpark already removed it in 2.0, I think we can get rid of saveAsParquetFile in sparkR in this release too.